docs/feature-gates.md

```diff
@@ -250,8 +245,24 @@ metadata:
     ipam.antrea.io/ippools: 'pool1'
 ```
 
+#### Data path change for this feature
+
+When `AntreaIPAM` is enabled, `antrea-agent` will connect the Node's network interface
+to the OVS bridge at startup, and it will detach the interface from the OVS bridge and
+restore its configurations at exit.
+Node may lose network connection when `antrea-agent`or OVS daemons are stopped
+unexpectedly, which can be recovered by rebooting the Node.
+`AntreaIPAM` Pods' traffic will not be routed by Node network stack.
+All traffic's destination MAC will be overwritten to Pod's MAC address when the
+destination IP is Pod IP on local Node and will be sent to destination OVS port directly,
+including `AntreaIPAM` Pods and regular `Subnet per Node` IPAM Pods, even they are not in
+the same subnets.
```

Comment:
Could you talk about cross Node traffic too?

Reply:
Added: `Inter-Node traffic will be sent from source Node to destination Node directly.`
Routing behavior is described above.